test_requirements.txt

```diff
@@ -9,6 +9,7 @@ pytest-capturelog
 pytest-watch
 pytest-xdist
 mock
+ipdb
```

Comment:
Ouch.  That probably snuck in when Shawn and I were pair programming.  I'll take responsibility for that though I never got it to work in docker on the mac.